src/tests/ftest/container/cont_rf_oclass_enforcement.yaml

```diff
@@ -0,0 +1,51 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+    - server-C
+    - server-D
+    - server-E
+    - server-F
+  test_clients:
+    - client-G
+timeout: 360
+server_config:
+  name: daos_server
+  servers:
+    targets: 2
+pool:
+  mode: 511
+  name: daos_server
+  scm_size: 1GB
+  svcn: 3
+  control_method: dmg
+  pool_query_timeout: 30
+container:
+  control_method: daos
+  akey_size: 5
+  dkey_size: 5
+  data_size: 5
+  object_qty: 10
+  record_qty: 10
+  cont_property: !mux
+    rf_0:
+      properties: rf:0
+    rf_1:
+      properties: rf:1
+    rf_2:
+      properties: rf:2
+    rf_3:
+      properties: rf:3
+    rf_4:
+      properties: rf:4
+rebuild:
```

Comment:
Exactly, why can't we keep the testing of rf0-rf4 with rebuild in rebuild/rebuild_container_rf.py and keep the testing of rf/oclass enforcement in cont_rf_oclass_enforcement.py? Why do we need to trigger rebuild here?

Reply:
If we do component test, we should focus on the simple configuration and simple verification as you described.
For system tests, as described in the JIRA ticket (DAOS-6268): after the container RF configured, launch traffic with rebuild to test the daos-server sustain.
Another consideration is: container RF is a new feature, it better to have more tests with traffic and rebuild, and not much overhead in running time.

Reply:
I do not disagree we need to test rf with rebuild but my point here is that flow is being tested in rebuild/rebuild_container_rf.py. 
I think we should also rethink the muxing of rf and oclass here. Can we do it differently to avoid overhead of variant setup/teardown?

Reply:
ok. 
I have removed the rebuild-tests after the RF verification from conatiner/cont_rf_oclass_enforcement test.
Please see the new commit.
Please note: 
Without rebuild test, we still need IO traffic with different oclass to test with different container-RF.  
In cont_rf_oclass_enforcement.yaml,  **rebuild:**  class is to parse the input oclass and rank for obj-write test.

Reply:
I think it would be good to remove the params "rebuild" and "rank" in this yaml file since they are not used and could be misleading. This can be cleaned up in a follow up PR.